EIP-3198 Draft -> Review

abdelhamidbakhta opened the pull request:
As discussed during ACD we change EIP-3198 Status to Review

eip-automerger commented:
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 3198 state from Draft to Review

MicahZoltu reviewed (changes requested):
Needs an Abstract.  Consider renaming Simple Summary to Abstract and then writing a new Simple Summary like "Adds an opcode that gives the EVM access to the block's base fee.".

----

Needs Rationale to be filled out.  I recommend at least discussing why you decided on the chosen gas cost.

----

Needs Security Considerations.  If there are no known security considerations mention that.

----

Recommend rewording Backward Compatibility to "There are no known backward compatibility issues with this opcode." to match phrasing in other EIPs.

abdelhamidbakhta commented:
> Needs an Abstract. Consider renaming Simple Summary to Abstract and then writing a new Simple Summary like "Adds an opcode that gives the EVM access to the block's base fee.".
> 
> Needs Rationale to be filled out. I recommend at least discussing why you decided on the chosen gas cost.
> 
> Needs Security Considerations. If there are no known security considerations mention that.
> 
> Recommend rewording Backward Compatibility to "There are no known backward compatibility issues with this opcode." to match phrasing in other EIPs.

Thanks for your review. I addressed your comments. Let me know if there is anything to change.

lightclient commented on EIPS/eip-3198.md:
```diff
@@ -31,11 +32,16 @@ Add a `BASEFEE` opcode at `(0x48)`, with gas cost `G_base`.
 
 ## Test Cases
 
+
```
Probably don't need this extra space.

MicahZoltu replied:
```suggestion
```
